functions/src/common/datastore.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2018 Google LLC
+ * Copyright 2022 Google LLC
```

Comment:
We should retain year the file was created. We can optionally add a range (`2018-2022`), but we haven't been doing that so far.

Reply:
I'll do the created year then.

But wait, how did this pick up the diff base? What I did was just create new ts file, copy content over and edit, and then delete js file. How did it know? 🧐

Reply:
It's clever!